### PR TITLE
update dalex with caret

### PR DESCRIPTION
**Pull request by kozaka93**
update a vignette DALEX with caret #43 

@jakwisn please review

### CODE REVIEW

**Comment by kozaka93** [comment]
fix typos in Comparison between breakdown, lime, shapley vignette #43

**Comment by jakwisn** [comment]
It is all right, did not see any typos or anything. I would only argue that when creating multiple explainers the verbose should be `false` so it will be better to look at.

File: vignettes/DALEX_caret.Rmd [hunk 1]
```diff
@@ -62,8 +66,8 @@ Validation dataset for the models is `apartmentsTest` data from the *DALEX* pack
 ```{r}
 data(apartmentsTest)
 
-explainer_regr_rf <- DALEX::explain(regr_rf, label="rf", 
-                                    data = apartmentsTest, y = apartmentsTest$m2.price,
+explainer_regr_rf <- DALEX::explain(regr_rf, label = "rf", 
+                                    data = apartmentsTest, y = apartmentsTest$m2.price, 
                                     colorize = FALSE)
 
 explainer_regr_gbm <- DALEX::explain(regr_gbm, label = "gbm",
```
Comment (jakwisn) [inline vignettes/DALEX_caret.Rmd:73]: maybe `verbose=FALSE` ?

Comment (kozaka93) [reply vignettes/DALEX_caret.Rmd:73]: Yes, good idea. I fix this.

**Pull request merged**